### DISCUSSION TITLE
Downloaded files default location?

### DISCUSSION THREAD

**Discussion by camusalf** [zotify-dev/zotify#54] [Q&A] [answered]
just downloaded two albums and don't know the location of the files, help pls

**Comment by camusalf** [comment, marked as answer]
nvm just found them in my default music folder

**Reply by arfelo1** [reply]
Where is it???!!!

**Reply by Hazeybxx** [reply]
anyone here able to help me out?

**Reply by arfelo1** [reply]
Found it. It is literally the default music folder as @camusalf says. Windows has a set of default folders: Downloads, Desktop, Documents, Images, Video and Music.

This "Music" folder should have another one inside it called "Zotify Music". This is the default download location by zotify.

If you want the root path, it should be:

C:\Users\{Your User}\Music\Zotify Music
